os-server/src/main/java/app/openschool/course/CourseRepository.java

```diff
@@ -29,4 +31,9 @@ public interface CourseRepository extends JpaRepository<Course, Long> {
   List<Course> getRandomSuggestedCoursesIgnoredExistingCourses(
       @Param(value = "limit") int limit,
       @Param(value = "existingCoursesIds") List<Long> existingCoursesIds);
+
+  @Query(
+          value = "SELECT * FROM learning_path lp  WHERE lp.mentor_id = :mentorId",
+      nativeQuery = true)
+  Page<Course> findCoursesByMentorId(@Param(value = "mentorId") Long mentorId, Pageable page);
 }
```

Comment:
After changings you don't need to use @Query and @Param already. The method will be just like this 
Page<Course> findCoursesByMentorId(Long mentorId, Pageable page);

Reply:
deleted